src/main/java/com/fasterxml/jackson/databind/deser/std/ThrowableDeserializer.java

```diff
@@ -196,4 +184,35 @@ public Object deserializeFromObject(JsonParser p, DeserializationContext ctxt) t
 
         return throwable;
     }
+
+    /*
+    /**********************************************************
+    /* Internal helper methods
+    /**********************************************************
+     */
+
+    /**
+     * Helper method to initialize Throwable
+     *
+     * @since 2.17
```

Comment:
Should have been....
```java
     * @since 2.16
```

I forgot to change the version 🙉.
May I ask you to change it when you have time, @cowtowncoder ?
PR feels like too much to take up everyone's attention

Reply:
should say 2.16.2

Reply:
> should say 2.16.2

Right 👍🏼